js/responsivegridsystem.js

```diff
@@ -0,0 +1,26 @@
+<!-- hide from old browsers

```

Comment:
Expected an identifier and instead saw '<'.<br>Expected an operator and instead saw '!'.<br>Expected an operator and instead saw '--'.<br>Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.<br>'hide' is not defined.<br>'from' is not defined.<br>'old' is not defined.<br>'browsers' is not defined.